controllers/glanceapi_controller.go

```diff
@@ -120,6 +121,27 @@ func (r *GlanceAPIReconciler) Reconcile(ctx context.Context, req ctrl.Request) (
 		return ctrl.Result{}, nil
 	}
 
+	// Get Glance Top level CR and fetch ExtraVolumes if defined
+	parentGlance := &glancev1.Glance{}
+
+	err = r.Client.Get(
+		context.TODO(),
+		types.NamespacedName{
+			Name:      glance.GetOwningGlanceName(instance),
+			Namespace: req.Namespace,
+		},
+		parentGlance)
+
+	// we don't need to fail here (or return an err) because it might happen
+	// that the Top Level resource is deleted but the underlying resources are
+	// not yet cleaned up: we need to wait for r.reconcileDelete being executed
+	// if the top-level resource (parentGlance) is empty, an empty array is
+	// assigned to instance.ExtraMounts
+	if err != nil {
+		r.Log.Info("Reconciling Service: Glance CR not present")
+	}
+	instance.ExtraMounts = parentGlance.Spec.ExtraMounts
```

Comment:
Should we check if the `err` is something other than `Not Found` and return immediately, just to be consistent with how we handle errors across the various operators?

Reply:
Done